src/styles/calendarDay.scss

```diff
@@ -0,0 +1,42 @@
+.calendar__day {
+  height: $squareSide;
+  width: $squareSide;
+
+  text-align: center;
+
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 330%;
+
+  background-color: #eee;
+  border: 1px solid black;
+
+  transition-duration: 0.5s;
```

Comment:
don't use it like that, because it's the same as transition: all 0.5s
It's too hard for the browser
Check this
![image](https://user-images.githubusercontent.com/91411692/223076112-d914e32c-7e65-42d7-9f80-856f0071213e.png)
also, create variable for transition duration
```suggestion
  transition: transform $transitionDuration, ...;
```